### PR TITLE
[B2BP-333] Added Cards section to Strapi

### PR DESCRIPTION
**Pull request by simosalsi**
Added Cards section to Strapi

#### List of Changes

- Created CardsItem
- Created Cards (using CardsItem)
- Added Cards to Page's "sections" field

#### Motivation and Context

Needed for duplicating SEND's website.

#### How Has This Been Tested?

Tested locally.

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by datalek — COMMENTED** [review]
I would add the [changeset](https://github.com/pagopa/b2b-portals/pull/137#issuecomment-1900466045)

File: apps/strapi-cms/src/components/sections/cards.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    },
+    "title": {
+      "type": "string",
+      "required": true
+    },
+    "subtitle": {
+      "type": "string"
+    },
+    "body": {
+      "type": "richtext"
```
Comment (kin0992) [inline apps/strapi-cms/src/components/sections/cards.json:26]: after [what discovered in another PR](https://github.com/pagopa/b2b-portals/pull/130#discussion_r1466175920), should we change this to `text`?

Comment (simosalsi) [reply apps/strapi-cms/src/components/sections/cards.json:26]: I don't think we should for now.

That issue has only presented itself for the BannerLink component (Editorial does the same and is fine for instance, Footer as well) so there's a good chance it's an isolated issue.

File: .changeset/thin-penguins-teach.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"strapi-cms": minor
+---
+
+Added Cards section to Strapi
```
Comment (kin0992) [inline .changeset/thin-penguins-teach.md:5]:
```suggestion
Add Cards section to Strapi
```

**Review by kin0992 — APPROVED** [review]

**Pull request merged**